### DISCUSSION TITLE
Not reading connections list

### DISCUSSION THREAD

**Discussion by maverusfxt** [codefori/vscode-ibmi#1210] [Q&A] [answered]
Yesterday I updated VSCode and the Code for IBM i extension and it no longer reads the list of connections and I can no longer connect to a system. 

Here is the error I continue to get: Error: Cannot read properties of undefined (reading 'getLastConnections'). A list of IBM i connections doesn't display and if I add one, it doesn't display either. I have tried many things to correct this, uninstalling and reinstalling the extension, removing the settings.json file between uninstalling and reinstalling, reverting to an earlier version of the extension. 

I also remember that when I last connected to the system it asked, if I remember correctly, about changing to use bash rather than IBM PASE (or vice-versa) and I clicked it to change it. Maybe this had an impact on my error.

Any insights on how to correct this are appreciated.

**Comment by worksofliam** [comment, marked as answer]
Ensure you are running Code for IBM i 1.7.12. We pushed a fix last night to solve it.

**Reply by maverusfxt** [reply]
Wahoo! Thanks! It broke yesterday and I didn't post until this morning. Thanks Liam!!
